### ISSUE TITLE
docmods -> class function

### ISSUE THREAD

**Issue by thareh** [bmx-ng/bmx-ng#60]
Hi,
One thing that's been bugging me for a long while is that the "Create" keyword is getting linked to "TThreadData.Create()" when the docs are generated. Only functions without a class should get linked yes? :)

Also, thanks for all the work & effort put into BlitzMax-NG! Highly appriciated! 👍

**Comment by woollybah** [comment]
Indeed, it would probably be better for `docmods` only to list "global" references, rather than what seems like everything.

Thanks :-)

**Comment by HurryStarfish** [comment]
Unfortunately, the syntax highlighting and F1 help of MaxIDE are very basic... it only knows to globally highlight specific words and doesn't understand anything about the language itself.
Ideally we would have something (whether MaxIDE or something else) that directly communicates with bcc, compiling and analyzing code in the background while it is being typed. That would be a prerequisite to allow it to do all the things you'd expect from an actual IDE: being able to tell where identifiers are defined and what their type is, showing compile errors in "real time", having an autocomplete feature, mouse-over help etc...
